### ISSUE TITLE
Animation does not work on mobile with React v17

### ISSUE THREAD

**Issue by adrianxadamn** [Stanko/react-animate-height#127]
Animation does not work on mobile with React v17

**Code example**
https://codesandbox.io/s/react-animate-height-basic-example-forked-4j23gl?file=/src/index.js:0-26

**Expected behavior**
On my Iphone 10, animation doesn't work compared to viewing on any browser on desktop

**Screenshots**

**Your Environment**
 - Version used:  "react-animate-height": "^2.1.2",
 - OS: iOS
 - Browser: Safari or Chrome

**Comment by Stanko** [comment]
Hey @adrianxadamn,
Any chance you could record what is happening as it works normally on my iphone 12 mini.
Cheers!

**Comment by Stanko** [comment]
I released version 3 yesterday, it is rewritten to hooks, maybe you can try it, as I can't replicate the issue.
Cheers!

**Comment by josuearrieta** [comment]
@adrianxadamn I can confirm I'm seeing the same behavior

**Comment by Stanko** [comment]
If v3.0.4 is still not working for you, can you please provide me with the codesandbox that doesn't work including the device and ios versions. Because I can't replicate it.

Using [this codesanbox](https://codesandbox.io/s/react-animate-height-basic-example-forked-3crf9z?file=%2Fsrc%2Findex.js) with version 3.0.4 works for me, on my iPhone 12 mini.

Here is the same sandbox, on iPhone X (using browserstack as don't have a physical device):

https://user-images.githubusercontent.com/776788/191068838-9656a1f5-418e-4197-a14d-a4955fdb3c9e.mov

And on iPhone 13 pro (on the iOS simulator):

https://user-images.githubusercontent.com/776788/191068231-575bd75e-3d04-4434-9474-d0320d49be3c.mp4


-----

**Comment by josuearrieta** [comment]
Sure. Mine is an iPhone XR (15.5).
This is how it looks using that codesandbox: 

https://user-images.githubusercontent.com/9735910/191290348-ada17fab-5fe7-490e-819f-df37de4e12bb.mp4

**Comment by Stanko** [comment]
I honestly don't know why it doesn't work :/ I just tested it on browser stack, iphone XR, ios 15.4

So it might be related to ios 15.5, but I don't how to pinpoint it. Any help is welcome!

iphone XR, ios 15.4:

https://user-images.githubusercontent.com/776788/191300302-92d22a83-a396-4ce2-8392-b3af26a86eef.mov

**Comment by josuearrieta** [comment]
So after digging a bit into my settings, I found out this:

![ReduceMotionSetting](https://user-images.githubusercontent.com/9735910/191317588-4741d43a-a1e1-40d7-804e-bc40075927ee.png)

There is this `Reduce Motion` option and I had it turned on. Once I turned it off, the animation was working as expected!
So this is not specific to an iOS version, or a device; but that particular setting :). 
Sorry for the inconvenience.
